crates/test_utils/src/newman_runner.rs

```diff
@@ -82,10 +95,94 @@ where
     Ok(())
 }
 
-pub fn generate_newman_command() -> ReturnArgs {
+// This function gives runner for connector or a module
+pub fn generate_runner() -> Result<ReturnArgs> {
+    let args = Args::parse();
+
+    let runner = match args.get_module_name() {
+        Some("users") => generate_newman_command_for_users(),
+        Some("connector") => generate_newman_command_for_connector(),
+        _ => generate_newman_command_for_connector(),
+    };
```

Comment:
Can we make this an `Option<Module>` instead, where `Module` is an enum like so:

```rust
enum Module {
    Connector,
    Users,
}
```